blogger/blogger.php

```diff
@@ -203,12 +203,12 @@ function blogger_send(App $a, array &$b)
 
 EOT;
 
-		Logger::log('blogger: data: ' . $xml, LOGGER_DATA);
+		Logger::log('blogger: data: ' . $xml, DATA);
```

Comment:
This is missing `Logger::` and the PR doesn't declare the `Logger` constants. You can still amend your commit and `git push --force`.